### ISSUE TITLE
Webinterface does not append changes

### ISSUE THREAD

**Issue by fe-z** [syncthing/syncthing#1192]
Hi, my problem is that the webinterface doesnt change my settings. The config.xml is read- and writeable by me (-rw-------). I can change my settings through this file, but its very inconvenient, This bug/misconfiguration happened after the newest update to v0.10.17 and is happening under archlinux on my desktop and my laptop.

**Comment by AudriusButkevicius** [comment]
Did you try refreshing the page?
Are there any errors in the syncthing log?
Are there any errors in the javascript log in the page?

**Comment by fe-z** [comment]
Yes i refreshed the page multiple times, still does not work. I also deinstalled syncthing, deleted all the config files and reinstalled it, the issue still appears. How do i log the page javascript? Also I'm new to GitHub, how do i paste the log properly? For now, here is the plain text from "STTRACE=model syncthing &> syncthing.log". 


[monitor] 2015/01/06 21:53:16.221352 monitor.go:104: INFO: Starting syncthing
[PLTTT] 2015/01/06 21:53:16.701044 main.go:401: INFO: syncthing v0.10.17 (go1.4 linux-amd64 default) builduser@jara 2015-01-04 19:18:14 UTC
[PLTTT] 2015/01/06 21:53:16.701142 main.go:402: INFO: My ID: PLTTTH7-DL33ELL-ITIG5HA-GI3UIQT-BCXL37V-3JC4BQN-BT37GLA-D5VFFQV
[PLTTT] 2015/01/06 21:53:16.705825 progressemitter.go:99: DEBUG: progress emitter: updated interval 5s
[PLTTT] 2015/01/06 21:53:16.706637 main.go:654: INFO: Starting web GUI on http://127.0.0.1:8080/
[PLTTT] 2015/01/06 21:53:16.706898 progressemitter.go:66: DEBUG: progress emitter: timer - looking after 0
[PLTTT] 2015/01/06 21:53:16.706917 progressemitter.go:82: DEBUG: progress emitter: nothing new
[PLTTT] 2015/01/06 21:53:18.139019 upnp.go:106: INFO: Starting UPnP discovery...
[PLTTT] 2015/01/06 21:53:18.838120 progressemitter.go:144: DEBUG: progress emitter: bytes completed for default: 0
[PLTTT] 2015/01/06 21:53:18.838143 model.go:417: DEBUG: model@0xc208061440 NeedSize("default"): 0 0
[PLTTT] 2015/01/06 21:53:19.932197 progressemitter.go:144: DEBUG: progress emitter: bytes completed for default: 0
[PLTTT] 2015/01/06 21:53:19.932220 model.go:417: DEBUG: model@0xc208061440 NeedSize("default"): 0 0
[PLTTT] 2015/01/06 21:53:24.139689 upnp.go:133: INFO: UPnP discovery complete (found 0 devices).
[PLTTT] 2015/01/06 21:53:24.139799 main.go:1134: INFO: Starting local discovery announcements
[PLTTT] 2015/01/06 21:53:24.140143 main.go:1139: INFO: Starting global discovery announcements
[PLTTT] 2015/01/06 21:53:24.141026 main.go:559: OK: Ready to synchronize default (read-write)
[PLTTT] 2015/01/06 21:53:24.141304 main.go:575: INFO: Device PLTTTH7-DL33ELL-ITIG5HA-GI3UIQT-BCXL37V-3JC4BQN-BT37GLA-D5VFFQV is "desktop-felix" at [dynamic]
[PLTTT] 2015/01/06 21:53:24.141389 puller.go:87: DEBUG: puller/default@0xc20826c500 starting
[PLTTT] 2015/01/06 21:53:24.142628 puller.go:212: DEBUG: puller/default@0xc20826c500 rescan
[PLTTT] 2015/01/06 21:53:24.143263 puller.go:226: DEBUG: puller/default@0xc20826c500 next rescan in 54.28299913s
[PLTTT] 2015/01/06 21:53:24.143293 puller.go:231: INFO: Completed initial scan (rw) of folder default
[PLTTT] 2015/01/06 21:53:24.257660 progressemitter.go:144: DEBUG: progress emitter: bytes completed for default: 0
[PLTTT] 2015/01/06 21:53:24.257685 model.go:417: DEBUG: model@0xc208061440 NeedSize("default"): 0 0
[PLTTT] 2015/01/06 21:53:25.141546 puller.go:138: DEBUG: puller/default@0xc20826c500 ignore patterns have changed, resetting prevVer
[PLTTT] 2015/01/06 21:53:25.141577 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:53:26.141738 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:53:27.141950 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0

[...]

[PLTTT] 2015/01/06 21:54:16.152725 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:17.152938 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:18.153141 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:18.426428 puller.go:212: DEBUG: puller/default@0xc20826c500 rescan
[PLTTT] 2015/01/06 21:54:18.426737 puller.go:226: DEBUG: puller/default@0xc20826c500 next rescan in 1m0.438713265s
[PLTTT] 2015/01/06 21:54:18.535506 progressemitter.go:144: DEBUG: progress emitter: bytes completed for default: 0
[PLTTT] 2015/01/06 21:54:18.535534 model.go:417: DEBUG: model@0xc208061440 NeedSize("default"): 0 0
[PLTTT] 2015/01/06 21:54:19.153413 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:20.153621 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0

[...]

[PLTTT] 2015/01/06 21:54:48.159266 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:49.159514 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[PLTTT] 2015/01/06 21:54:50.159759 puller.go:148: DEBUG: puller/default@0xc20826c500 skip (curVer == prevVer) 0
[monitor] 2015/01/06 21:54:50.651757 monitor.go:130: INFO: Signal 2 received; exiting

**Comment by AudriusButkevicius** [comment]
You add triple backticks (`) and close with triple backticks.
The log is not really relevant in this case, the javascript console would be more relevant.
It depends on the browser, so google around.
